frontend/app/src/modules/balances/exchanges/use-exchanges.ts

```diff
@@ -102,6 +103,16 @@ export function useExchanges(): UseExchangesReturn {
     }
   };
 
+  const fetchSelectedExchangeBalances = async (refresh = false, exchangeLocation=""): Promise<void> => {
```

Comment:
having refresh as a parameters here does not make sense since it's only called once with value `true` we can just put `ignoreCache: true` instead. 

Also having the `exchangeLocation` default to an empty string does not make sense since we should always call it with a specific exchange